test/test_bregman.jl

```diff
@@ -20,8 +20,8 @@ function obj(x)
     return fun, grad
 end
 
-opt = bregman_options(maxIter=200, progTol=0, verbose=2)
-sol = bregman(obj, 1 .+ randn(N2), opt)
+opt = bregman_options(maxIter=200, progTol=0, verbose=2, antichatter=false) # anti chatter now only works with real number
```

Comment:
This needs to raise a proper error if attempted. Why isn't it supported?

Reply:
Anti-chatter needs to record number of -1/1 when the gradient fluctuates. If things are complex then there is no -1/1

Reply:
Yes there is no `sign` in complex which is why there is a different  thresholding function for this case so the anti-chatter should follow it and use the angle instead of sign but I think it should follow

Reply:
hmm in anti-chatter there is a sum of -1/1. How do we do it with angle? A sum of unit vectors with different angles? And then do inner product to calculate the scaling of gradient?

Reply:
Probably something like that. I don't mind it leaving it as a todo and see if can make something like that work, would assume it would by summing the angles. But would adda warning or error if someone tries it so that it's readable not a lengthy error trace

Reply:
gotcha make sense